### PR TITLE
Send messages!

### PR DESCRIPTION
**Pull request by ashanan**

### CODE REVIEW

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -11,10 +11,28 @@ def index
...
 
     @messages_logged = messages.count
+    @messages_authorized = messages.find_all { |message| message.authorized }.count
+    @messages_sent = messages.find_all { |message| message.sent }.count
 
     @distinct_zips = Message.select(:zip).distinct.length
   end
+
+  def send_messages
+    unsent_messages = Message.where({sent: false, authorized: true})
```
Comment (mattbaker) [inline app/controllers/admin_controller.rb:28]: what did `authorized` mean again?

**Comment by mattbaker** [comment]
Honestly, your guess is as good as mine on this one, but it seems ok? I've never used mailers or any of this stuff.

Think we could test-run with a couple messages in the DB just to verify before we hit the button ferreal?

**Comment by ashanan** [comment]
That they clicked through on the confirmation email.

File: app/controllers/admin_controller.rb (hunk 1 above)
Comment (ashanan) [reply app/controllers/admin_controller.rb:28]: That they clicked through on the confirmation email.

**Comment by ashanan** [comment]
I've run it locally and it's worked 🤷‍♂️

**Comment by mattbaker** [comment]
🚢!

**Pull request merged**